Makefile.vc

```diff
@@ -0,0 +1,266 @@
+#
+# Options:
+# machine=Win32|Win64
+# prefix=\path\to\install
+#
+# If no machine is specified, the makefile uses the systems
+# PROCESSOR_ARCHITECTURE.
+#
+# Compiling:  nmake -f Makefile.vc
+# Installing: nmake -f Makefile.vc prefix=\path\to\x install
+#
+
+VERSION=0.1.5
+
+# Compiler, linker, librarian, and other tools
+CC  = cl
+CXX = cl
+LD  = link
+AR  = lib
+
+!IFDEF MACHINE
+!IF "$(MACHINE)" == "Win64"
+TARGET_FLAGS = /D__x86_64__
```

Comment:
Right.
`link` and `lib` are still required since nmake does not define [macros](https://msdn.microsoft.com/en-us/library/d7ahf12s(v=vs.110).aspx) for them. I was not able to create a DLL with `cl` and `/LD`.

The `PROCESSOR_ARCHITECTURE` condition is required for 64-bit builds.  The VS (2015 at least) environment does not define `__x86_64__`.